api/utils.js

```diff
@@ -25,14 +25,20 @@ module.exports = {
         let milli = seconds.substr(seconds.indexOf('.') + 1, 3);
         timeParts[2] = timeParts[2].substr(0, timeParts[2].indexOf('.'));
         let date = new Date(dateParts[0], dateParts[1], dateParts[2], timeParts[0], timeParts[1], timeParts[2], milli);
-        date = sumHour(date, -3);
+        date = this.sumHour(date, -3);
         return date;
     },
 
     setTime : function(date, hour, minute) {
         return new Date(date.getFullYear(), date.getMonth(), date.getDate(), hour, minute, 0, 0);
     },
 
+    sumHour : function(date, numberOfHours) {
```

Comment:
Nothing broke without this function, so do we really need it? :thinking:

Reply:
it breaks once a year hehe

Reply:
it is used on convertToDate, I don't know how it doesn't break anything